var/spack/repos/builtin/packages/sirius/package.py

```diff
@@ -79,17 +80,11 @@ class Sirius(CMakePackage, CudaPackage, ROCmPackage):
 
     variant("shared", default=True, description="Build shared libraries")
     variant("openmp", default=True, description="Build with OpenMP support")
-    variant(
-        "boost_filesystem",
-        default=False,
-        description="Use Boost filesystem for self-consistent field method "
-        "mini-app. Only required when the compiler does not "
-        "support std::experimental::filesystem nor std::filesystem",
-    )
```

Comment:
Shouldn't this stay for older versions? `when="@:7:4.3`?

Reply:
Sorry, my bad. Indeed it should. The `CMAKE_CXX_STANDARD` is still 14 for older versions.

Reply:
After double checking I think it is not needed and can be removed. The oldest compiler we use is gcc9. It supports std::filesystem::experimental. I've removed the boost dependency here: 
https://github.com/electronic-structure/SIRIUS/commit/ea581c1ef8fc5f6ee0e33131f144af38bac8eee0

Previously we checked if std::experimental::filesystem is available and used a namespace alias. `+boost_filesystem` only required to be set manually in case the compiler did not even support experimental filesystem.

Reply:
Thanks for looking into this @simonpintarelli. @toxa81 do you want to revert to the removal of the variant? Sorry for the extra work!